src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -0,0 +1,102 @@
+    </When>
+    <Otherwise>
+      <!-- No specific NuGet targets to sequence after, run before targets that consume references -->
+      <PropertyGroup>
+        <ResolveAssemblyReferencesDependsOn>ImplicitlyExpandNETStandardFacades;$(ResolveAssemblyReferencesDependsOn)</ResolveAssemblyReferencesDependsOn>
+        <PrepareResourcesDependsOn>ImplicitlyExpandNETStandardFacades;$(PrepareResourcesDependsOn)</PrepareResourcesDependsOn>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <PropertyGroup Condition="'$(ImplicitlyExpandNETStandardFacades)' == 'true'">
+    <!-- Ensure this runs before conflict resolution since the added files may cause conflicts -->
+    <HandlePackageFileConflictsDependsOn>ImplicitlyExpandNETStandardFacades;$(HandlePackageFileConflictsDependsOn)</HandlePackageFileConflictsDependsOn>
+  </PropertyGroup>
+
+  <UsingTask TaskName="GetDependsOnNETStandard" AssemblyFile="$(MicrosoftNETBuildExtensionsTasksAssembly)" />
+  <UsingTask TaskName="NETBuildExtensionsError" AssemblyFile="$(MicrosoftNETBuildExtensionsTasksAssembly)" />
+  <Target Name="ImplicitlyExpandNETStandardFacades"
+          AfterTargets="$(ImplicitlyExpandNETStandardFacadesAfter)">
+
+    <ItemGroup>
+      <_candidateNETStandardReferences Include="@(Reference);@(_ResolvedProjectReferencePaths)" />
+      <_inboxNETStandardFolders Include="$(TargetFrameworkDirectory)" />
+    </ItemGroup>
+
+    <!-- determine if NETStandard.dll is supported inbox for this TargetFramework -->
+    <PropertyGroup Condition="'$(NETStandardInbox)' == ''">
+      <NETStandardInbox Condition="Exists('%(_inboxNETStandardFolders.Identity)\netstandard.dll')">true</NETStandardInbox>
+    </PropertyGroup>
+
+    <!-- determine if any references depend on NETStandard -->
+    <GetDependsOnNETStandard Condition="'$(DependsOnNETStandard)' == '' AND '$(NETStandardInbox)' != 'true' AND '@(_candidateNETStandardReferences)' != ''" 
+                             References="@(_candidateNETStandardReferences)">
+      <Output TaskParameter="DependsOnNETStandard" PropertyName="DependsOnNETStandard" />
+    </GetDependsOnNETStandard>
+
+    <!-- prevent using an older SDK version with NETStandard2.0 references -->
+    <PropertyGroup>
+      <_usingOldSDK Condition="'$(UsingMicrosoftNETSdk)' != 'true' AND ('$(TargetFramework)' != '' OR '$(TargetFrameworks)' != '')">true</_usingOldSDK>
+    </PropertyGroup>
+    <NETBuildExtensionsError Condition="'$(DependsOnNETStandard)' == 'true' AND '$(NETStandardInbox)' != 'true' AND '$(_usingOldSDK)' == 'true'"
+                             ResourceName="UnsupportedSDKVersionForNetStandard20"/>
+    
+    <!-- if any reference depends on netstandard and it is not inbox, add references and implementation assemblies for netstandard2.0  -->
+    <ItemGroup Condition="'$(DependsOnNETStandard)' == 'true' AND '$(NETStandardInbox)' != 'true'">
+      <_NETStandardLibraryNETFrameworkReference Condition="'$(_TargetFrameworkVersionWithoutV)' &gt;= '4.7'"
```

Comment:
RAR will remove the DLL if it becomes inbox in the framework.  We don't remove DLLs within subsequent out-of-band releases.  This is existing logic that's just been reimplemented from the package.